.github/workflows/push.yml

```diff
@@ -630,6 +630,12 @@ jobs:
       - name: Login to Amazon ECR
         id: login-ecr
         uses: aws-actions/amazon-ecr-login@v1
+      - name: Scan image
+        uses: Azure/container-scan@v0
+        with:
+          image-name: ${{ env.SERVICE_IMAGE_NAME }}:${{ env.TAG }}
+          severity-threshold: CRITICAL
+          run-quality-checks: true
```

Comment:
Přemýšlím teda jestli to stačí až takto při releasu. Nebo mám raději udělat samostatný job který pojede paralelně s testy a bude se spouštět při každým pushi?

Reply:
kludne asi vzdy, k to nebude trvat `10m`